### PR TITLE
fix cadl sample

### PR DESCRIPTION
**Pull request by dw511214992**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ options:
     "emitter-output-dir": "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
     "package-mode": "dataplane"
   "@azure-tools/cadl-csharp":
-    namespace: Azure.Contoso.Widgetmanager
+    namespace: Azure.Template.Contoso
```
Comment (raych1) [inline documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml:24]: Why only change namespace for csharp? how about java?  

Another inconsistent thing is the `emitter-output-dir` of typescript, can it be changed to use value of namespace?

File: documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl [hunk 2]
```diff
@@ -53,7 +53,7 @@ interface Widgets {
   listWidgets is ResourceList<
     Widget,
     {
-      parameters: StandardListQueryParameters & SelectQueryParameter;
+      parameters: StandardListQueryParameters;
```
Comment (raych1) [inline documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl:56]: Is the reason of removing this due to SDK generation failure?

File: documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml (hunk 1 above)
Comment (dw511214992) [reply documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml:24]: namespace `Azure.Contoso` is not registered and .Net sdk pipeline reports error. So we need to reuse an existing test namespace.

File: documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl (hunk 2 above)
Comment (dw511214992) [reply documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl:56]: .Net sdk emitter doesn't support it.

**Review by raych1 — APPROVED** [review]

**Pull request merged**